nrf_wifi/fw_if/umac_if/inc/default/fmac_structs.h

```diff
@@ -99,6 +99,11 @@ struct raw_rx_pkt_header {
  * the UMAC IF layer needs to invoke for various events.
  */
 struct nrf_wifi_fmac_callbk_fns {
+	/** Callback function to be called when RPU recovery is required. */
+	void (*rpu_recovery_callbk_fn)(void *os_vif_ctx,
```

Comment:
Should this also be protected by `#if defined(CONFIG_NRF_WIFI_RPU_RECOVERY)` ?

Reply:
This was intentional, if that isn't defined it will be assigned NULL.